docs/semgrep-ci/running-semgrep-ci-with-semgrep-app.md

```diff
@@ -328,11 +329,107 @@ SEMGREP_PR_ID="44"
+
+1. In Bitbucket, select the repository where you want Semgrep to run in CI.
+1. Click **Repository settings** > **Access tokens**.
+1. Click **Create Repository Access Token**.
+1. Create a token name. Note: As Semgrep comments to your PRs, use **Semgrep** as the name for this token.
+1. Under **Scopes**, select the **Read**, and **Write** permissions for pull requests.
+1. Click **Create**.
+1. Copy the value of the repository access token, and then click **Close**.
+1. Click **Repository variables**.
+1. Create a new workspace variable by entering **PAT** into the **Name** field, and then paste the workspace access token in the **Value** field.
+1. Enable the **Secured** option, and then click **Add**.
+
+Create and add a `SEMGREP_APP_TOKEN` to establish the communication between your repository and the Semgrep Cloud Platform by following these steps:
+
+1. Create `SEMGREP_APP_TOKEN` by following the [Creating a `SEMGREP_APP_TOKEN`](/semgrep-ci/running-semgrep-ci-with-semgrep-app/#creating-a-semgrep_app_token) documentation. Copy the value of the token created in the Semgrep Cloud Platform.
+1. In Bitbucket, select the repository where you want Semgrep to run in CI.
+1. Click **Repository settings** > **Access tokens**.
+1. Click **Create Workspace Access Token**.
+1. Create a new workspace variable by entering **SEMGREP_APP_TOKEN** into the **Name** field, and then paste the value of the token in the **Value** field.
+1. Enable the **Secured** option, and then click **Add**.
+
+##### Adding Semgrep to your Bitbucket CI pipeline for PR comments
+
+To receive comments on PRs, add the following code to your `bitbucket-pipelines.yml` file:
+
+```yaml
+image: atlassian/default-image:latest
+
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          name: 'Run Semgrep scan in PR branch'
+          image: returntocorp/semgrep
+          script:
+            - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
+            - export SEMGREP_APP_URL="https://semgrep.dev"
+            - export SEMGREP_BASELINE_REF="origin/main"
+            - git fetch origin "+refs/heads/*:refs/remotes/origin/*"
+            - export BITBUCKET_TOKEN=$PAT
+            - semgrep ci
+```
+
+:::note
+Substitute branch names and exported tokens (for example `main`, `$PAT`) according to your workspace or repository settings.
+:::
+
+As a result, your Bitbucket repositories are now part of the [Projects](https://semgrep.dev/orgs/-/projects) page.
+
+:::info
+- Test Semgrep PR comments by submitting a test code from a rule in your [Rule board](https://semgrep.dev/orgs/-/board) that is in the **Comment** column.
+- Only rules that are in the **Comment** column of your [Rule board](https://semgrep.dev/orgs/-/board) create the PR comments.
```

Comment:
Related comment above but this would be Comment or Block column on the Rule Board.

Reply:
Originally I was including Block column also, but I removed it based on our docs. I will test this again and edit it, thanks!

Reply:
Yeah, so for the comment column, Semgrep will leave a comment but will **not** block on CI. If the rule is in the Block column, Semgrep will leave a comment **and** block the pipeline build.

Reply:
Thank you!

Reply:
@pangngan so what do you think about:
```md
Only rules that are in the **Comment** and **Block** column of your [Rule board](https://semgrep.dev/orgs/-/board) create the PR comments. Rules from the **Block** will also block the PR pipeline. To ublock the pipeline, detected code needs to be fixed. 
```